src/algo/depth-to-rgb-calibration/valid-scene.cpp

```diff
@@ -676,7 +684,12 @@ bool optimizer::is_scene_valid(input_validity_data* data)
     }
     AC_LOG(DEBUG, "    " << _yuy.section_map.size() << " not suppressed");
```

Comment:
The messages are now less useful... movement detection will now happen twice.
Either we specify which one were doing so we have context, or we remove this message and others like it and only leave the conclusion?